ADOPTERS.md

```diff
@@ -0,0 +1,43 @@
+## End-users
+
+Known end-users include: Bloomberg, Bytedance, Pinterest, Square, and Uber.
+
+SPIFFE and SPIRE are being used by numerous other companies, both large and small, to build higher layer products and services. This includes Decipher Technology Studios, F5 Networks, HashiCorp, Intel, Google, IBM, Tigera, VMware, and many others.
+
+## Ecosystem
+
+SPIFFE and SPIRE have integrations available with a number of open-source projects. The list includes:
+
+Kubernetes
+Docker
+Envoy
+Istio
+NGINX
+Ghosttunnel
+Knox	
+Consul
+Hamlet
+
+## Case Studies/User Stories
```

Comment:
In the style of other CNCF project repositories, this file is intended to be a source file consumed both by machines, and also people navigating the repo and not the website. Rather than link from this file to another page, the page can get that data from here.